airflow/utils/state.py

```diff
@@ -41,6 +41,21 @@ class State(object):
     UPSTREAM_FAILED = "upstream_failed"
     SKIPPED = "skipped"
 
+    task_states = (
+        SUCCESS,
+        RUNNING,
+        FAILED,
+        UPSTREAM_FAILED,
+        UP_FOR_RETRY,
+        QUEUED,
+    )
```

Comment:
Probably not. I wasn't aware of the separate tuple when I implemented "scheduled". So the GUI didnt consider them by coincidence. Now, cause it is in state.py and defined as task_states, it gets confusing as the states do not fully cover all states.